packages/grafana-ui/src/components/Table/BarGaugeCell.tsx

```diff
@@ -35,32 +35,25 @@ export const BarGaugeCell: FC<TableCellProps> = (props) => {
   }
 
   const displayValue = field.display!(cell.value);
+  const customFieldOptions: TableFieldOptions = field.config.custom;
+  const cellOptions = customFieldOptions.cellOptions;
 
   // Set default display mode
   let barGaugeMode: BarGaugeDisplayMode = BarGaugeDisplayMode.Gradient;
+  let valueMode = BarGaugeValueMode.Color;
 
-  // Support deprecated settings
-  const usingDeprecatedSettings = field.config.custom.displayMode !== undefined;
-
-  // If we're using the old settings format we read the displayMode directly from
-  // the cell options
-  if (usingDeprecatedSettings) {
-    if (
-      (field.config.custom && field.config.custom.cellOptions.displayMode === TableCellDisplayMode.Gauge) ||
-      (field.config.custom && field.config.custom.cellOptions.displayMode === BarGaugeDisplayMode.Lcd)
-    ) {
```

Comment:
I could not really understand this, why are we checking `cellOptions.displayMode` here? we have never had such a custom option

Reply:
Might be a mistake on my part there 😬